build.sh

```diff
@@ -491,12 +492,20 @@ if (( ${NUMARGS} == 0 )) || hasArg go; then
 fi
 
 # Build the cuvs Java bindings
+RUN_JAVA_TESTS=""
+for arg in "$@"
+do
+  if [ "$arg" == "--run-java-tests" ]; then
+    RUN_JAVA_TESTS="--run-java-tests"
+  fi
+done
+
```

Comment:
```suggestion
```

This corresponds to my suggestion a few lines below to reuse `hasArg` that's already defined in this script, instead of introducing a new variable and a `for` loop over all arguments.

Reply:
fixed